go/libraries/doltcore/sqle/dtablefunctions/dolt_log.go

```diff
@@ -688,14 +689,16 @@ func (itr *logTableFunctionRowIter) Next(ctx *sql.Context) (sql.Row, error) {
 	var commitHash hash.Hash
 	var commit *doltdb.Commit
 	var optCmt *doltdb.OptionalCommit
+	var meta *datas.CommitMeta
+	var height uint64
 	var err error
 	for {
-		commitHash, optCmt, err = itr.child.Next(ctx)
+		commitHash, optCmt, meta, height, err = itr.child.Next(ctx)
 		if err != nil {
 			return nil, err
 		}
 		ok := false
-		commit, ok = optCmt.ToCommit()
+		commit, ok = optCmt.ToCommit() // TODO: unnecessary? the matchFn already does this
```

Comment:
Do we not need this then? Feels like we should resolve this TODO before checking this in.